frontend/index.html.ejs

```diff
@@ -39,15 +14,51 @@
 
   <script type="text/javascript">
     <% if (htmlWebpackPlugin.options.production) { %>
-    window['NOMAD_ENDPOINT'] = window.location.host;
+    window['NOMAD_PRELOAD_ENDPOINT'] = (window.location.host + window.location.pathname).replace(/\/$/, '');
     <% } else { %>
-    window['NOMAD_ENDPOINT'] = window.location.hostname + ':' + (NOMAD_ENDPOINT_PORT || 3000);
+    window['NOMAD_PRELOAD_ENDPOINT'] = window.location.hostname + ':' + (NOMAD_ENDPOINT_PORT || 3000);
     <% } %>
-
     var configScript = document.createElement('script');
-    configScript.setAttribute('src', '//' + NOMAD_ENDPOINT + '/config.js');
+    configScript.setAttribute('src', '//' + NOMAD_PRELOAD_ENDPOINT + '/config.js');
```

Comment:
couldn't we just always ask for `./config.js` without and path, then it would be whatever path you are on now + `config.json` which would satisfy the requirement in the Go server i think :)